os-autoinst-obs-auto-submit

```diff
@@ -21,7 +21,17 @@ XMLSTARLET=$(command -v xmlstarlet || true)
 get_submit_target_extra_latest_version() {
     # awk iterates through all the fields in the match to avoid fixed position
     # and it should match with the part which starts with `submit_target_extra_project`+`:`
-    latest_submit_target_extra_version=$(osc dists | grep "$submit_target_extra_project" | awk -v prj="$submit_target_extra_project" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tail -n 1)
+    version_sorted=$(osc dists | grep "$submit_target_extra_project" | awk -v prj="$submit_target_extra_project" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tac)
+    local rc=0
+    latest_submit_target_extra_version=''
+    for version in $version_sorted; do
+        # dont need the XML output for checking existence.
+        # if $prj does not exist return 404
+        if (osc meta prj "${version}:Update" > /dev/null 2>&1); then
```

Comment:
```suggestion
        if osc meta prj "${version}:Update" > /dev/null 2>&1; then
```

I assume you confused languages. In bash this would be a subshell but there is no need to execute this in a subshell

Reply:
> But what for? It's never read anywhere else

ok let me take a look. because at that point I dont remember what i have changed, why, how....

Reply:
> But what for? It's never read anywhere else

I dont know what for. it was removed.